<!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>The Family Tree of Life</title>
  
  <meta property="og:site_name" content="The Panda's Thumb" />
  <meta property="og:type" content="article" />
  <meta property="og:title" content="The Family Tree of Life" />
  <meta property="og:url" content="https://felsenst.github.io/PT/archives/2014/10/the-family-tree.html" />
  <meta property="og:image" content="https://felsenst.github.io/PT/media/roar.jpg" />
  <meta property="article:author" content="Emily Thompson" />
  <meta property="article:published_time" content="2014-10-28T14:10:46-07:00" />


  <link rel="stylesheet" type='text/css' href="/PT/css/normalize.css" />
  <link rel="stylesheet" type='text/css' href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic,700italic|Source+Code+Pro:400,700" />
  <link rel="stylesheet" type='text/css' href="/PT/css/gridism.css" />
  <link rel="stylesheet" type='text/css' href="/PT/css/style.css" />
  <link rel="stylesheet" type='text/css' href="/PT/css/github.css" />
  <link rel="icon" type="image/x-icon" href="/PT/favicon.ico" />
  <script type="text/javascript" src="//code.jquery.com/jquery-3.1.1.min.js"></script>
  <script type="text/javascript" src="https://use.fontawesome.com/e6a24c6dde.js"></script>
  <script type="text/javascript" async src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>  
  <script src="/PT/js/local.js"></script>

</head>


<body class="wrap">
  <div class="grid page-fill"><div></div></div>
<header class="topbar">
  <div class="grid no-gutters">
    <div class="banner unit whole">
		  <img src="/PT/media/banner.jpg" alt="The Panda's Thumb" />
    </div>
    <nav class="main-nav unit whole align-center">
      <ul>
<li><a href="/PT/" title="Main Page"><i class="fa fa-home" aria-hidden="true"></i>&nbsp;Main</a></li>
<li><a href="/PT/archives/index.html" title="Archives"><i class="fa fa-book" aria-hidden="true"></i>&nbsp;Archives</a></li>
<li><a href="#" title="Search is Unavailable"><i class="fa fa-search" aria-hidden="true"></i>&nbsp;Search</a></li>
</ul>

    </nav>
  </div>
</header>


    <section class="blog">
    <div class="grid">
      <div class="unit whole">
        <article class="blogpost">
  <header>
	  <h1><a href="/PT/archives/2014/10/the-family-tree.html">The Family Tree of Life</a></h1>
	  <div class="byline"><i class="fa fa-user-circle" aria-hidden="true"></i> By Emily Thompson</div>
	  <div class="attrline"><i class="fa fa-clock-o" aria-hidden="true"></i>
	  <span><time pubdate="pubdate" datetime="2014-10-28T14:10:46-07:00">October 28, 2014 14:10 MST</time></span>
	  </div>
   </header>
  <p><em>In the next few weeks, we’ll be posting a series of articles for the general public focused on understanding how species are related and how genomic data is used in research. We start with a background on phylogenetic trees.</em></p>

<p>Imagine you could go back in time and meet your great grandmother or even your great-great-great-great-great grandmother, when they were your age. Would they look like you? Or would they look more like your siblings or cousins? Maybe you would all look a little different. Scientists try to figure out how the distant ancestors of apes, other animals, plants, and all organisms living today looked and behaved, much in the same way that people use a family tree to trace their ancestry.</p>

<figure>
<img src="/PT/uploads/2014/primate-family-tree-780x520_0.gif" alt="primate-family-tree-780x520_0.gif" width="450" height="300" style="float:right;" />
<figcaption>
The common ancestor of great apes lived about 18 million years ago.
Source: Smithsonian National Museum of Natural History http://humanorigins.si.edu/evidence/genetics

</figcaption>
</figure>

<p>In evolutionary biology scientists use a type of tree called a “phylogenetic tree” to organize the history of how species descended from common ancestors. The closer two species are to a common ancestor on the phylogenetic tree, the more closely the two are related.</p>

<p>Take the phylogenetic tree of primates, for example. The common ancestor of apes lived about 18 million years ago. But over time, this one group branched off to form many different species, including humans, which have their own separate branch on this tree.</p>

<p>How did so many unique species develop from one ancestor? New branches formed by a process known as divergence. When groups of ancient organisms became geographically isolated from one another, either through migration or geologic events like earthquakes, each group began to develop its own unique set of physical attributes. Sometimes, by chance, a change in a characteristic enabled an individual to survive better in its environment and produce more offspring.</p>

<p>Perhaps individuals in one group with larger arms were better able to break open the hard-shelled fruits that were common in one region, while some individuals in another group had the ability to travel more easily through tall trees that offered protection from predators. Whatever the reason may have been, selection favored genetic differences that improved survival. Over time, this gradual process of isolation and selection produced distinct species, which in turn branched into more species.</p>

<p>The end result of divergence is many species, related in a tree-like fashion, and we display these relationships using phylogenetic trees. Scientists now use increasingly sophisticated methods to determine how species were related and build phylogenetic trees. In the past, scientists built these trees simply by comparing physical traits, like how many limbs an organism has or whether it has a tail. But with the recent surge in fast and affordable gene sequencing technologies, researchers today can directly compare species’ DNA to determine how they are related.</p>

<p>But analyzing entire genomes, with billions of DNA base pairs, presents its own unique set of challenges, and researchers often struggle to determine if the DNA differences they find between species are truly significant or are simply due to common variability.  As computer software and statistical analysis become more adept at handling these challenges, our understanding of species’ relationships could change — providing exciting new insights into our family tree of life.</p>

<p><em>Check back next week when we discuss the differences between studying small and large datasets, and the challenges associated with big data analysis.  This series is supported by NSF Grant #DBI-1356548 to RA Cartwright.</em></p>

</article>

      </div>

    </div>
  </section>


  <footer>
<div class="grid">
<table bgcolor="204517">
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>
  <div align="left">
    <p>&nbsp;&nbsp;To see earlier posts, select the Archives at the top of this page</p>
    <p></p>
  </div>
  <div id="RecentComments" class="dsq-widget">
      <div align="center">
       <h2>Recent Comments</h2>
       <P>
       To see the comment in context of the discussion click on the text that indicates how long ago the comment was posted, such as "2 hours ago".  Then wait for the post and then the comments to load.
      <P>
    </div>
    <div>
     <script type="text/javascript" src="https://the-pandas-thumb.disqus.com/recent_comments_widget.js?num_items=15&hide_mods=0&hide_avatars=0&avatar_size=32&excerpt_length=100"></script>
    </div>
  </div>
</td></tr>
  <tr><td></td><td><div class="unit whole">
      <p>Copyright &copy; The Panda&#8217;s Thumb and original authors &mdash; Content provided under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons BY-NC-ND License 4.0</a>.</p>
    <p></p>
    </div>
    </td></tr>
  </table>
  </div>
  </footer>

  

</body>

</html>


